news/_posts/2022-12-23-playtest-20221223.md

```diff
@@ -0,0 +1,113 @@
+---
+kind: article
+title: "Playtest 20221223"
+author: "PunkPun"
+created_at: 2022-12-22 23:00 +0000
+disqus_id: "playtest-20221223"
+permalink: "/news/playtest-20221223/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Fixed a few pathfinder crashes
+* Fixed a few Red Alert missions
+* Fixed carryalls in rare cases destrying their cargo
+* Added caryalls to the D2K spectator economy statistics
+* Added new maps to the D2K map pool
+* A few other minor fixes
+
+We expect this will be our final playtest before a full release on New Year's, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).
```

Comment:
New Year's *Day*? So first of January. Or New Year's Eve? I wouldn't mention this at all but just write
```suggestion
We expect this will be our final playtest before a full release, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).
```